tests/src/Interop/MarshalAPI/ReadWrite/ReadWriteObject.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
```

Comment:
Do we have any rules about when we test corelib via coreclr/tests vs. corefx tests? The latter makes it easier to test compat with other platforms and also keeps everything in one place. Of course coreclr is convenient..